app/pages/ParticipatingGroups/groups.ts

```diff
@@ -0,0 +1,38 @@
+export default [
+  {
+    id: 'cyprusjs',
```

Comment:
Can we fetch this info from https://raw.githubusercontent.com/cyprus-developer-community/home/main/groups.json ?

Reply:
Yeap. That's the last bit we need. I put it there to remind me to ask in the channel for the answer. @PatrickHeneise do you know who I need to ask for a small description of each group, as well as the official website link?